clients/spark/core/src/main/scala/io/treeverse/clients/ApiClient.scala

```diff
@@ -194,6 +190,18 @@ class ApiClient private (conf: APIConfigurations) {
     } else ""
   }
 
+  def getCommit(repoName: String, commitID: String): api.model.Commit = {
+    commitsApi.getCommit(repoName, commitID)
+  }
+  
+  /** Query lakeFS for a URL to the metarange of commitID of repoName and
+   *  translate that URL to use an appropriate Hadoop FileSystem.
+   */
+  def getMetaRangeURL(repoName: String, commitID: String): String = {
+    val commit = commitsApi.getCommit(repoName, commitID)
+    getMetaRangeURL(repoName, commit)
+  }
+
```

Comment:
I prefer to keep overloads together.  Something like this, perhaps...

```suggestion
  /** Query lakeFS for a URL to the metarange of commitID of repoName and
   *  translate that URL to use an appropriate Hadoop FileSystem.
   */
  def getMetaRangeURL(repoName: String, commitID: String): String = {
    val commit = commitsApi.getCommit(repoName, commitID)
    getMetaRangeURL(repoName, commit)
  }

  def getCommit(repoName: String, commitID: String): api.model.Commit = {
    commitsApi.getCommit(repoName, commitID)
  }
```

Reply:
Done